src/main/java/testapp/core/UserAccount.java

```diff
@@ -0,0 +1,56 @@
+package testapp.core;
+
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.Key;
+
+import java.io.Serializable;
+
+public class UserAccount implements Serializable {
+    private Key userKey;
```

Comment:
cuz I if I stored as long value then I should parse that value into key, that why I decide to stored as a key object